pagopa/io-app
# chore: [IOBP-591] Removed 401 failure handlers to the payments requests features

## Short description
This PR removes the 401 failure handlers from the entire `payments` features to let the `withRefreshApiCall` handle this kind of status failure.

## List of changes proposed in this pull request
- Added a check to not perform the failure if the response has a response with status 401

## How to test
Run the `io-dev-app-server` enabling LV with a short expiration time and then try to perform any action on the new payments sections (such as onboarding, checkout, etc.). You should be able to renew the session without showing a failure screen after renewing it.
